projects/pepe-dex/index.js

```diff
@@ -0,0 +1,86 @@
+const sdk = require("@defillama/sdk");
+
+const ADDRESSES = require("../helper/coreAssets.json");
+const { nullAddress, sumTokensExport } = require("../helper/unwrapLPs");
+const { scriptEvaluate, assetBalance, sumTokens } = require("../helper/chain/waves");
+
+// DEX actually doesn't store WAVES on its account
+// but stakes it to our WAVES staking contract
+// so TVL calculation is based on product of SWAVES/WAVES rate with SWAVES balance
+async function wavesTVL(api) {
+  const WAVES_VAULT_CONTRACT = "3PCSFoiiY4eUcLJNzSfMLy5jHEMtcHduT3M";
+  const SWAVES_STAKING_CONTRACT = "3PDPzZVLhN1EuzGy4xAxjjTVkawKDLEaHiV";
+  const SWAVES_ASSET_ID = "YiNbofFzC17jEHHCMwrRcpy9MrrjabMMLZxg8g5xmf7";
+  const SWAVES_RATE_DECIMALS = 1e12;
+
+  const balanceResp = await assetBalance(
+    WAVES_VAULT_CONTRACT,
+    SWAVES_ASSET_ID
+  );
+
+  const sWavesBalance = balanceResp.balance;
+  const evaluateResp = await scriptEvaluate(
```

Comment:
please dont count staked sWAVES, I think this is double counting the underlying WAVES